packages/debian-tests/rules

```diff
@@ -24,7 +24,7 @@ override_dh_auto_build:
 	dh_auto_build -- CUDA=$(CUDA) lib exes
 
 override_dh_shlibdeps:
-	dh_shlibdeps -Xapiperf -Xcopybw -Xcopylat -Xsanity
+	dh_shlibdeps -Xgdrcopy_apiperf -Xgdrcopy_copybw -Xgdrcopy_copylat -Xgdrcopy_sanity
 
 override_dh_auto_install:
```

Comment:
does dpk automatically picks the symlinked files?

Reply:
Yes.
```
$ dpkg -c ./gdrcopy-tests_2.3-1_amd64.Ubuntu20_04+cuda12.0.deb
drwxr-xr-x root/root         0 2021-07-23 11:59 ./
drwxr-xr-x root/root         0 2021-07-23 11:59 ./usr/
drwxr-xr-x root/root         0 2021-07-23 11:59 ./usr/bin/
-rwxr-xr-x root/root     30904 2021-07-23 11:59 ./usr/bin/gdrcopy_apiperf
-rwxr-xr-x root/root     34992 2021-07-23 11:59 ./usr/bin/gdrcopy_copybw
-rwxr-xr-x root/root     34992 2021-07-23 11:59 ./usr/bin/gdrcopy_copylat
-rwxr-xr-x root/root    120928 2021-07-23 11:59 ./usr/bin/gdrcopy_sanity
drwxr-xr-x root/root         0 2021-07-23 11:59 ./usr/share/
drwxr-xr-x root/root         0 2021-07-23 11:59 ./usr/share/doc/
drwxr-xr-x root/root         0 2021-07-23 11:59 ./usr/share/doc/gdrcopy-tests/
-rw-r--r-- root/root      5029 2021-07-23 11:59 ./usr/share/doc/gdrcopy-tests/README.Debian.gz
-rw-r--r-- root/root       181 2021-07-23 11:59 ./usr/share/doc/gdrcopy-tests/changelog.Debian.gz
-rw-r--r-- root/root      1268 2021-07-23 11:59 ./usr/share/doc/gdrcopy-tests/copyright
lrwxrwxrwx root/root         0 2021-07-23 11:59 ./usr/bin/apiperf -> gdrcopy_apiperf
lrwxrwxrwx root/root         0 2021-07-23 11:59 ./usr/bin/copybw -> gdrcopy_copybw
lrwxrwxrwx root/root         0 2021-07-23 11:59 ./usr/bin/copylat -> gdrcopy_copylat
lrwxrwxrwx root/root         0 2021-07-23 11:59 ./usr/bin/sanity -> gdrcopy_sanity
```